### PR TITLE
fix flaky spaces test

### PR DESCRIPTION
**Pull request by Dzuming**
- fix flaky spaces test

### CODE REVIEW

File: docker-based-ror-dev-env/runInDocker.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ fi
 
 docker run --rm $DIND_OPTIONS $DOCKER_RUN_OPTIONS \
   -e ROR_ACTIVATION_KEY="$ROR_ACTIVATION_KEY" \
+  -e I_UNDERSTAND_AND_ACCEPT_KBN_PATCHING=yes \
```
Comment (coutoPL) [inline docker-based-ror-dev-env/runInDocker.sh:33]: let's just hardcode it in the Dockerfile in this line:
```
$INVOKE_ROR_TOOLS patch && \
```

Comment (Dzuming) [reply docker-based-ror-dev-env/runInDocker.sh]: Done ✔️

File: environments/elk-ror/images/kbn/Dockerfile [hunk 2]
```diff
@@ -4,8 +4,10 @@ FROM docker.elastic.co/kibana/kibana:${KBN_VERSION}
 
 ARG KBN_ROR_FILE
 ARG ROR_ACTIVATION_KEY
+ARG I_UNDERSTAND_AND_ACCEPT_KBN_PATCHING
 
 ENV ROR_ACTIVATION_KEY=$ROR_ACTIVATION_KEY
+ENV I_UNDERSTAND_AND_ACCEPT_KBN_PATCHING=$I_UNDERSTAND_AND_ACCEPT_KBN_PATCHING
```
Comment (coutoPL) [inline environments/elk-ror/images/kbn/Dockerfile:10]: now, it seems we don't need this one

File: environments/elk-ror/docker-compose.yml [hunk 3]
```diff
@@ -47,6 +47,7 @@ services:
         KBN_VERSION: $KBN_VERSION
         KBN_ROR_FILE: $KBN_ROR_FILE
         ROR_ACTIVATION_KEY: $ROR_ACTIVATION_KEY
+        I_UNDERSTAND_AND_ACCEPT_KBN_PATCHING: yes
```
Comment (coutoPL) [inline environments/elk-ror/docker-compose.yml:50]: and this one

**Review by coutoPL — APPROVED** [review]
minor comments left

File: environments/elk-ror/images/kbn/Dockerfile (hunk 2 above)
Comment (Dzuming) [reply environments/elk-ror/images/kbn/Dockerfile]: Done ✔️

File: environments/elk-ror/docker-compose.yml (hunk 3 above)
Comment (Dzuming) [reply environments/elk-ror/docker-compose.yml]: Done ✔️

**Pull request merged**